Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla43527.cs

```diff
@@ -39,5 +39,24 @@ public TestPage()
 				};
 			}
 		}
+
+#if UITEST && __WINDOWS__
```

Comment:
The original problem was on Windows, but we should test for it on all platforms to prevent regressions. So we don't need the `__WINDOWS__`.

Reply:
Removed Windows preprocessor